integrations/gtag/HISTORY.md

```diff
@@ -1,3 +1,9 @@
+1.1.0 / 2020-11-02
```

Comment:
```suggestion
1.0.1 / 2020-11-02
```

Just a bug fix so I think it should be third digit: https://docs.npmjs.com/about-semantic-versioning